tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_clone_livelist_condense_races.ksh

```diff
@@ -114,4 +120,5 @@ set_tunable32 LIVELIST_CONDENSE_SYNC_PAUSE 1
 disable_race LIVELIST_CONDENSE_SYNC_CANCEL
 delete_race LIVELIST_CONDENSE_SYNC_CANCEL
 
+
```

Comment:
tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_clone_livelist_dedup.ksh doesn't have 2 lines before last log_pass, what is the consistent one?

Reply:
And poof, the extra line is gone.